### PR TITLE
Update `Game mode/osu!taiko`

### PR DESCRIPTION
**Pull request by SherlockWh0**
My first contribution! Please give feedback.
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-

### CODE REVIEW

File: wiki/Game_mode/osu!taiko/en.md [hunk 1]
```diff
@@ -24,24 +24,22 @@ Alternatively, click on the `Mode` button and select `osu!taiko`.
 
 For players with prior *Taiko no Tatsujin* experience:
 
-- No chibi dancers at the bottom (must be storyboarded)
-- The health bar must be at least 50% full to pass the song (which is not really obvious using the default skin; please look in the [Skin Compendium](https://osu.ppy.sh/community/forums/topics/180864) for a decent osu!taiko skin)
+- No PuchiChara at the bottom (must be storyboarded)
```
Comment (TicClick) [inline wiki/Game_mode/osu!taiko/en.md:27]: the change makes it far harder to understand what it is, please revert (also, IMO the specific name isn't important here in context of the game)

File: wiki/Game_mode/osu!taiko/en.md [hunk 2]
```diff
@@ -24,24 +24,22 @@ Alternatively, click on the `Mode` button and select `osu!taiko`.
...
+- Background difference
+  - A video or static background image-only beatmap will take up the lower portion of the screen
+  - If the map has been storyboarded, the storyboard will take over the bottom of the screen
+For newcomers of osu!taiko mode, the screen is split into two different sections; the upper section contains the gameplay elements while the lower section contains an image, video, or storyboard for the beatmap. In the upper section, there is the health bar, which unlike other game modes, starts empty and must be filled up to at least 50% full or half-full to pass the difficulty. At the top-middle part below the health bar, a taiko drum is located at the left side and a moving conveyor belt which carries the hit objects from right to left passing through the judgement circle near the taiko drum. Above the taiko drum is an animated taiko mascot (otherwise known as a pippidon or don) which will react based on kiai sections, combo milestones, and notes missed during kiai time along with a scrolling background slider that changes colour when notes are missed. Lastly, there is the usual score, accuracy, and timer of the beatmap at the upper-right.
```
Comment (TicClick) [inline wiki/Game_mode/osu!taiko/en.md:34]: while I can see some things changed for better, it's still a stream of text readable poorly enough to make me either backtrace or double the efforts required to following. not your fault, but that gives your change little to no benefit

**Review by TicClick — CHANGES_REQUESTED** [review]
I think it's good enough since you didn't aim for cleaning up the whole article -- please address my comments, fix CI errors, and we can get this merged

File: wiki/Game_mode/osu!taiko/en.md (hunk 2 above)
Comment (TicClick) [reply wiki/Game_mode/osu!taiko/en.md:34]: (if anything this doesn't require a reply -- more of a side comment I made without realising neither action would help the article in its current shape anyway)

File: wiki/Game_mode/osu!taiko/en.md (hunk 1 above)
Comment (TicClick) [reply wiki/Game_mode/osu!taiko/en.md:27]: no, the dancers bit

**Review by peppy — DISMISSED** [review]
Looks like an overall improvement 👍

File: wiki/Game_mode/osu!taiko/en.md (hunk 2 above)
Comment (SherlockWh0) [reply wiki/Game_mode/osu!taiko/en.md]: made it quite a bit shorter and easier to read

**Review by SherlockWh0 — COMMENTED** [review]
added changes

File: wiki/Game_mode/osu!taiko/en.md [hunk 3]
```diff
@@ -76,11 +77,11 @@ The default controls for osu!taiko are:
 | Red | Left click(L) | `X` (L) / `C` (R) | Flat surface of the drum |
 | Blue | Right click(L) | `Z` (L) / `V` (R) | Outer surface of the drum |
 
-The placement of in-game cursor does not matter when playing.
+The placement of the in-game cursor does not matter when playing.
 
-If [Relax](/wiki/Gameplay/Game_modifier/Relax) mod was used, the score judgement only account for hit timing only (automatically rectify wrong colour hit with correct colour).
+If [Relax](/wiki/Gameplay/Game_modifier/Relax) mod is used, the score judgement only accounts for hit timing (the mod automatically hits the correct color).
```
Comment (Walavouchey) [inline wiki/Game_mode/osu!taiko/en.md:82]: the wiki uses british english

```suggestion
If [Relax](/wiki/Gameplay/Game_modifier/Relax) mod is used, the score judgement only accounts for hit timing (the mod automatically hits the correct colour).
```

**Review by Walavouchey — APPROVED** [review]
force-pushed to squash commits

**Pull request merged**